File: Solutions in Java/Reverse Words in a String.java
/**
* @author : Anmol Agrawal
*/
class Solution {
    public String reverseWords(String s) {
        String strs[] = s.trim().split(" ");
        String o = "";
        for(int i = strs.length - 1; i >= 0; i--){
            String x = strs[i];
            if(x.length() > 0){
                o = o + x + " ";
            }
        }
        return o.trim();
    }
}

File: Solutions in Javascript/ReverseWordsInAString.js
/**
 * @author : Anmol Agrawal
 */
/**
 * @param {string} s
 * @return {string}
 */
var reverseWords = function(s) {
    let strs = s.trim().split(" ");
    let o = "";
    for(let i = strs.length - 1; i >= 0; i--){
        let x = strs[i];
        if(x.length > 0){
            o = o + x + " ";
        }
    }
    return o.trim();
};